schedule/yast/lvm_raid1/lvm+raid1_sle15_x86_64.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name:           lvm+raid1@64bit
+description:    >
+  Validation of partitioning for raid1 on lvm
+  Installation of RAID1 using expert partitioner.
+vars:
+  RAIDLEVEL: 1
+  LVM: 1
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/skip_module_registration
+  add_on_product:
+    - installation/add_on_product/skip_install_addons
+  suggested_partitioning:
```

Comment:
In the adaption in [PR#16218](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16218) this will be the default already.

Reply:
here we have 3 cases, expert, guided and suggested(like default) but we can think later if we need three keys or overwrite one key called 'partitioning', perhaps is too soon, so we are good I think.